AirCasting/CreateSessionViews/WifiPopupView.swift

```diff
@@ -12,15 +12,14 @@ import SystemConfiguration.CaptiveNetwork
 
 struct WifiPopupView: View {
     
-    @State private var isWifiNameDisplayed: Bool = false
+    @State private var isSSIDTextfieldDisplayed: Bool = false
     @Environment(\.presentationMode) private var presentationMode
-    @State var currentNetworkSSID: String?
     @Binding var wifiPassword: String
     @Binding var wifiSSID: String
 
     var body: some View {
         VStack(alignment: .leading, spacing: 30){
-            if isWifiNameDisplayed {
+            if isSSIDTextfieldDisplayed || wifiSSID.isEmpty {
                 providePasswordTitle
                 createTextfield(placeholder: "Wi-Fi name", binding: $wifiSSID)
                 createTextfield(placeholder: "Password", binding: $wifiPassword)
```

Comment:
Should we hide user's password when he is entering it? We could use secure field https://developer.apple.com/documentation/swiftui/securefield
Actually same could be done at log in

Reply:
Both signup and login views have SecureFied.
I can change it here but I have some doubts about it. It's not that easy to uncover input in a secure field. That's why I chose to use the normal text field. We don't check if the password is correct and if it's incorrect the AB3 won't be configured correctly and the user won't get any measurements. We won't be able to inform users why the session is empty.

Reply:
My bad, I got confused and thought that the password was visible when I was signing in, but I see that it's not. When I was signing in I noticed that we didn't use keyboard for emails there, and it could be a better experience for user if we did. But that's an irrelevant issue here. :) This might be good idea to show the password to the user in this case.

Reply:
I already changed the keyboard for email address, thanks :)